src/screens/Connections/Connections.tsx

```diff
@@ -22,7 +22,11 @@ import EmptyState from './components/EmptyState/EmptyState';
 import Switch from './components/Switch/Switch';
 import {flights} from '../../data';
 
-const Connections = () => {
+type Props = {
+  navigation: any;
+};
+
+const Connections = ({navigation}: Props) => {
```

Comment:
Again, we should type correctly our files, using any just in last of the last cases, until there we have to know how type our props.

```suggestion
type Props = NativeStackScreenProps<
  {
    Connections: undefined;
  },
  'Profile'
>;

const Connections = ({navigation}: Props) => {

```


pickup a documentation for more info: https://reactnavigation.org/docs/typescript


btw we need to install the @react-navigation/native-stack to use the NativeStackScreenProps